### PR TITLE
k8s: consistent variable names and updated samples

### PR DESCRIPTION
**Pull request by ryane**
- [x] Installs cleanly on a fresh build of most recent master branch
- [x] Upgrades cleanly from the most recent release
- [x] Updates documentation relevant to the changes

* tested GCE but not digitalocean or openstack
* Uses consistent naming for kube* variables
* Updates digitalocean and openstack samples with Kubernetes support

### CODE REVIEW

**Comment by stevendborrelli** [comment]
@BrianHicks any comments on this one?

**Comment by BrianHicks** [comment]
LGTM

File: terraform/vsphere/main.tf [hunk 1]
```diff
@@ -215,12 +215,12 @@ output "worker_ips" {
...
-output "kube_worker_ips" {
-  value = "${join(\",\", vsphere_virtual_machine.mi-kube-worker-nodes.*.network_interface.ip_address)}"
+output "kubeworker_ips" {
+  value = "${join(\",\", vsphere_virtual_machine.mi-kubeworker-nodes.*.network_interface.ip_address)}"
 }
 
-output "kube_worker_ips" {
-  value = "${join(\",\", vsphere_virtual_machine.mi-kube-worker-nodes.*.network_interface.ip_address)}"
+output "kubeworker_ips" {
+  value = "${join(\",\", vsphere_virtual_machine.mi-kubeworker-nodes.*.network_interface.ip_address)}"
```
Comment (siddharthist) [inline terraform/vsphere/main.tf]: Should there be two of this line?

Comment (BrianHicks) [inline terraform/vsphere/main.tf]: hmm, probably not

**Comment by siddharthist** [comment]
@BrianHicks I rebased, but poorly. Could you double check my work and merge if it's ready?

**Comment by BrianHicks** [comment]
what did you rebase? It doesn't look like anything changed except our discussion.

**Comment by siddharthist** [comment]
@BrianHicks The vsphere terraform config had changed in master, I overrode it with ryan's changes. Good, I think I did it all correctly.

**Comment by sehqlr** [comment]
LGTM

**Pull request merged**